instrumentation/mongo/mongo-3.1/library/README.md

```diff
@@ -0,0 +1,49 @@
+# MongoDB library instrumentation
+
+This package contains the library to help instrument MongoDB Client.
+
+## Quickstart
+
+### Dependencies
+
+Replace OPENTELEMETRY_VERSION with the [latest release](https://central.sonatype.com/search?q=g:io.opentelemetry.instrumentation++a:opentelemetry-mongo-3.1&smo=true).
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-ktor-2.0</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```gradle
+implementation("io.opentelemetry.instrumentation:instrumentation:opentelemetry-mongo-3.1:OPENTELEMETRY_VERSION")
+```
+
+## Usage
+
+The instrumentation is initialized by passing a `MongoTelemetry::newCommandListener()` to the `MongoClientSettings` builder. You must set the `OpenTelemetry` to use with the feature.
+
+```kotlin
+val openTelemetry: OpenTelemetry = initializeOpenTelemetryForMe()
+val mongoTelemetry = MongoTelemetry.builder(openTelemetry).build()
+
+val settings = MongoClientSettings.builder()
+    .applyConnectionString(connectionString)
+    .addCommandListener(mongoTelemetry.newCommandListener())
+    .build()
+
+// With Reactive Streams
+val client = MongoClients.create(settings)
+```
```

Comment:
can you change the example from Kotlin to Java? thx

Reply:
[Fixed](https://github.com/open-telemetry/opentelemetry-java-instrumentation/compare/ba398725736b220bfd5854f95b899deef6705497..37cd7121a20773e604b15429c1ec733834455835).